expo/expo
# [expo-updates][android] Fix manifest headers in multipart responses

# Why

Implement spec changes from https://github.com/expo/expo/pull/15495.

# How

Send all headers down to manifest construction. Manifest part headers take precedence over the root response headers.

# Test Plan

1. Run tests.
2. Load EAS update project which now is serving this format (still see unrelated asset downloading error though)
